### ISSUE TITLE
"examples" fail 1.4.1 Use of Color in WCAG 2.1 document

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#570]
While "Notes" contain the text "Notes" in the green "message boxes", the yellow "Examples" only have a color to distinguish them. I would expect this to be consistent and also have the examples to contain the visual text "Examples"

Can be seen in https://www.w3.org/TR/WCAG21/ 

Quick link to one of the examples right beneath Notes...: https://www.w3.org/TR/WCAG21/#glossary

**Comment by michael-n-cooper** [comment]
[Examples are formatted by Respec](https://github.com/w3c/respec/wiki/example). Respec will put in a heading if heading text was provided, but not if it was not. If you think a generic "Example:" heading should be provided even when heading text wasn't provided, I suggest you [file that enhancement request on Respec](https://github.com/w3c/respec/issues). If you think a manual heading should be provided for every example in WCAG, then you could file a pull request on WCAG 2.1.
